src/utils/os_zephyr.c

```diff
@@ -5,7 +5,8 @@
  * See README for more details.
  */
 
-#include <zephyr/posix/time.h>
+#include <time.h>
+
 #include <zephyr/posix/sys/time.h>
```

Comment:
can this also be removed now?

Reply:
In the future, they will likely also include other ISO C headers that are multi-purposes for POSIX too. E.g.

* stdio.h

The nice thing about all of this, is that (by 4.3.0) we will not need to use a `<zephyr/posix/..>` prefix for any POSIX headers, because the build system will know to take those from either the toolchain's libc (for libcs that do support POSIX) or zephyr (for libc's that do not support POSIX).